pyqtgraph/parametertree/Parameter.py

```diff
@@ -20,9 +23,17 @@ def registerParameterItemType(name, itemCls, parameterCls=None, override=False):
     """
     global _PARAM_ITEM_TYPES
     if name in _PARAM_ITEM_TYPES and not override:
-        raise Exception("Parameter item type '%s' already exists (use override=True to replace)" % name)
-
-    parameterCls = parameterCls or Parameter
+        raise Exception(
```

Comment:
Can we raise something other than a bare exception here?

Reply:
wait, this shows as resolved, but still shows as a bare exception, some git mixup?

Reply:
Ha! I fixed an identical problem elsewhere in the file. This one's fixed now, too.